caps/static/js/main.js

```diff
@@ -272,3 +272,31 @@ var trackEvent = function(eventName, params) {
 $('.details-accordion').on('click', function(){
     $(this).siblings('.details-accordion[open]').removeAttr('open');
 });
+
+$('.nlzh-landing-page').on('click', 'a[href]', function(e){
```

Comment:
`.nzlh-landing-page` surely?

Reply:
God dammit. I never get these letters the right way round. Good spot!